test/inductor/test_torchbind.py

```diff
@@ -0,0 +1,63 @@
+# Owner(s): ["module: functorch"]
```

Comment:
`test/dynamo_expected_failures/TestTorchbind.test_torchbind_inductor`

New failure here - expected ?

Reply:
Yeah, this is expected since dynamo can't trace the torchbind objects yet.